client/src/main/java/com/github/jsappz/lamegame/engine/graph/Texture.java

```diff
@@ -20,8 +24,13 @@ public Texture(String fileName) throws Exception {
             IntBuffer w = stack.mallocInt(1);
             IntBuffer h = stack.mallocInt(1);
             IntBuffer channels = stack.mallocInt(1);
-
-            buf = stbi_load(fileName, w, h, channels, 4);
+            try (InputStream is = getClass().getClassLoader().getResourceAsStream(fileName)) {
```

Comment:
Note this code is going to be pretty expensive. It gets the resource input stream, reads all the bytes into a byte array, then copies all the bytes from a byte array into a byte buffer. I didn't see another good option, but might be worth a follow up I will let @JReistad decide

Reply:
will add todo for looking into this